tests/integration_test.rs

```diff
@@ -32,7 +32,7 @@ fn test_web_and_worker_procfile() {
                 assert_eq!(log_output.stdout, "this is the web process!\n");
             });
 
```

Comment:
Looking at the existing integration tests, it seems there are quite a few gaps in coverage - eg there are none checking behaviour for passing args etc, compared to libcnb-test which tests several permutations of changing command vs args:
https://github.com/heroku/libcnb.rs/blob/e9072af1b71bfcd23ac8cf70f3620299a7eebaef/libcnb-test/tests/integration_test.rs#L443-L513

I know this is pre-existing, but I wonder if some extra tests might be useful given the questions around the command/args handling?

Reply:
Yeah, makes sense. Done in [abe484e](https://github.com/heroku/procfile-cnb/pull/205/commits/abe484ea6aab9ff6139e3edae849bb13103fd39b).